test/vcr_python_wrapper.py

```diff
@@ -23,8 +23,8 @@ def body_json_l2_matcher(r1, r2):
     elif r1.headers.get('content-type') == 'multipart/form-data' and r2.headers.get('content-type') == 'multipart/form-data':
         if r1.body is None or r2.body is None:
             return r1.body == r2.body
-        body1 = sorted(r1.body.split(b'&'))
-        body2 = sorted(r2.body.split(b'&'))
+        body1 = sorted(r1.body.replace(b'~', b'%7E').split(b'&'))
+        body2 = sorted(r2.body.replace(b'~', b'%7E').split(b'&'))
```

Comment:
this feels like a terrible hack, but is the easiest way we can ensure our tapes work across all python versions we support.

@mdellweg can you bring me some shower gel please?

Reply:
@evgeni I'd say you should go for Sterillium.

Well if it helps for now.

Reply:
The whole function is a terrible hack to begin with. So i think it's gonna be ok.

Reply:
Can we run `encode()` instead?

Reply:
I think `str.encode` can only convert between ["real" encodings](https://docs.python.org/3/library/codecs.html#standard-encodings) and not URL style ones.